lms/static/certificates/sass/_config.scss

```diff
@@ -0,0 +1,37 @@
+// ------------------------------
+// Open edX Certificates: Config
+
+// About: variable and configuration overrides
+
+// #VARIABLES
+
+// ------------------------------
+// #VARIABLES
+// ------------------------------
+$em-base:       16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
```

Comment:
@talbs Can you give myself and @dsjen more context here. Why are you having to set the em-base here?